packages/ui-core/src/components/paginated-table/components/header/header.component.tsx

```diff
@@ -3,27 +3,68 @@ import { HeaderGroup } from 'react-table';
 
 import { Typography } from '../../../../types';
 import { HeaderCell } from '../header-cell';
-import { Head } from './header.styles';
+import { Head, StyledTH } from './header.styles';
```

Comment:
Could You please rename it to `StyledTableHeader` as we do not use uppercase convention in repository.